server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStore.java

```diff
@@ -142,13 +143,15 @@ public XodusServerIssueStore(Path backupDir, Path workDir) throws IOException {
       entityStore.registerCustomPropertyType(txn, IssueSeverity.class, new IssueSeverityBinding());
       entityStore.registerCustomPropertyType(txn, RuleType.class, new IssueTypeBinding());
       entityStore.registerCustomPropertyType(txn, Instant.class, new InstantBinding());
+      entityStore.registerCustomPropertyType(txn, HotspotReviewStatus.class, new HotspotReviewStatusBinding());
     });
 
     entityStore.executeInExclusiveTransaction(afterInit);
   }
 
   private PersistentEntityStore buildEntityStore() {
     var environment = Environments.newInstance(xodusDbDir.toAbsolutePath().toFile(), new EnvironmentConfig()
+        .setEnvMonitorTxnsTimeout(5000)
```

Comment:
What is the reason for this?

Reply:
woops, I used it for debugging an error at some point. Will remove, good catch!